src/pocketmine/Player.php

```diff
@@ -2791,10 +2791,10 @@ public function handleDataPacket(DataPacket $packet){
 						}else{
 							$thrownPotion->spawnToAll();
 						}
+					}elseif($item->getId() == Item::BOW){
+						$this->setDataFlag(self::DATA_FLAGS, self::DATA_FLAG_ACTION, true);
+						$this->startAction = $this->server->getTick();
 					}
-
-					$this->setDataFlag(self::DATA_FLAGS, self::DATA_FLAG_ACTION, true);
-					$this->startAction = $this->server->getTick();
```

Comment:
I'd like to observe that this is more likely to break things than to fix them. Why should this only be true for a bow? This will be executed for a bow regardless.